src/net_processing.cpp

```diff
@@ -2837,6 +2841,9 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             if (fReachable)
                 vAddrOk.push_back(addr);
         }
+        if (rate_limiting_triggered) {
+            LogPrint(BCLog::NET, "Rate limiting an address message from peer=%d\n", pfrom.GetId());
```

Comment:
Couldn't this just be in the `if` where `rate_limiting_triggered` is being set and thus remove the need for the bool?

Reply:
Good idea, done.